### PR TITLE
Clarify docstring for split-paths

### PR DESCRIPTION
**Pull request by Sohalt**
Please answer the following questions and leave the below in as part of your PR.

- [ ] This PR corresponds to an [issue with a clear problem statement](https://github.com/babashka/babashka/blob/master/doc/dev.md#start-with-an-issue-before-writing-code).

- [ ] This PR contains a [test](https://github.com/babashka/babashka/blob/master/doc/dev.md#tests) to prevent against future regressions

- [ ] I have updated the [CHANGELOG.md](https://github.com/babashka/fs/blob/master/CHANGELOG.md) file with a description of the addressed issue.

### CODE REVIEW

File: src/babashka/fs.cljc [hunk 1]
```diff
@@ -773,7 +773,8 @@
   (-> path split-ext last))
 
 (defn split-paths
-  "Splits a string joined by the OS-specific path-seperator into a vec of paths."
+  "Splits a path list given as a string joined by the OS-specific path-seperator into a vec of paths.
```
Comment (borkdude) [inline src/babashka/fs.cljc:776]: type: separator

File: src/babashka/fs.cljc [hunk 2]
```diff
@@ -773,7 +773,8 @@
...
+  On UNIX systems, the separator is ':', on Microsoft Windows systems it is ';'."
```
Comment (borkdude) [inline src/babashka/fs.cljc:777]: We have a var for that in this namespace, maybe just mentioned that?

Comment (Sohalt) [reply src/babashka/fs.cljc:777]: Mentioning the var means I'll have to look at the var, which "hides" the actual value behind a java static field, so I then have to look at the documentation for that (where I copied part of the docstring from). In my opinion the docstring should make immediately clear, that this is not aboutbb splitting a path into its components, but splitting a list of paths into individual paths (it's hinted at by the function name: split-path*s*, not split-path, but I still tripped). If the docstring just mentions the var, I'd most likely assumed the var is `/` on UNIX and `\` on Windows (without looking at the var).

**Comment by borkdude** [comment]
Thanks!

**Pull request merged**